amethyst_assets/src/storage.rs

```diff
@@ -36,6 +36,12 @@ pub struct AssetStorage<A: Asset> {
     unused_handles: MsQueue<Handle<A>>,
 }
 
+impl<A: Asset> Default for AssetStorage<A> {
+    fn default() -> Self {
+        Self::new()
```

Comment:
The standard seems to be reverse, except if deriving Default, but who knows.

Reply:
There's not really a standard. Clippy recommends it the way I prefer, rust-lang does it the other way around. This pattern is used in Specs, has been used so far in Amethyst AFAIK and you also used it in some places, so I thought unifying it would be best.